tools/gbdk_infinity.diff

```diff
@@ -1,6 +1,6 @@
-diff -ru SDK.orig/gbz80-gb/2-1-5/lib/crt0.s SDK/gbz80-gb/2-1-5/lib/crt0.s
---- SDK.orig/gbz80-gb/2-1-5/lib/crt0.s	1999-10-17 22:02:46.000000000 -0700
-+++ SDK/gbz80-gb/2-1-5/lib/crt0.s	2016-07-25 14:14:31.603316904 -0700
+diff -ru sdk.orig/gbz80-gb/2.1.5/lib/crt0.s sdk/gbz80-gb/2.1.5/lib/crt0.s
+--- sdk.orig/gbz80-gb/2.1.5/lib/crt0.s	1999-10-17 22:02:46.000000000 -0700
++++ sdk/gbz80-gb/2.1.5/lib/crt0.s	2016-07-25 14:14:31.603316904 -0700
```

Comment:
Yeah, just checked Windows package of GBDK. My goal is to move game compilation process to standard Makefile (so we can drop do*.bat files) and to add shell (unix) + batch (windows) scripts to automate toolchain preparing (unless we'll be able to move to newer GBDK version or to GBDK-N fork). 
I guess it will be needed to make two diffs or make another script to patch the diff file...